snapcraft.yaml

```diff
@@ -590,12 +594,12 @@ parts:
 
   nvidia-container:
     source: https://github.com/NVIDIA/libnvidia-container
-    source-commit: f23e5e55ea27b3680aef363436d4bcf7659e0bfc # v1.17.4
+    source-commit: d26524ab5db96a55ae86033f53de50d3794fb547 # v1.17.7
     source-depth: 1
     source-type: git
     plugin: make
     build-environment:
-      - GIT_TAG: "1.17.4" # Enables source-depth: 1, should match git tag without "v" prefix.
+      - GIT_TAG: "1.17.6" # Enables source-depth: 1, should match git tag without "v" prefix.
```

Comment:
Once #809 is merged it should also be cherry-picked.

Reply:
very good thanks, done